app/graphql/mutations/update_dynamic_field.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+class Mutations::UpdateDynamicField < Mutations::BaseMutation
+  argument :string_key, ID, required: true
+  argument :display_label, String, required: true
+  argument :sort_order, Integer, required: true
+  argument :is_facetable, Boolean, required: true
```

Comment:
Same here, I don't think the `is_*` fields should be required.

Reply:
I think we also don't need `required: true` for sort_order here, since there will be many cases where we're either ONLY updating dynamic field info without regard to sort_order.

And also, now that I think about it, there will be times when we're only updating sort order.  That may be a separate mutation though, one that factors in the relative order of fields and field groups.